sdk/go/pulumi-language-go/main.go

```diff
@@ -167,6 +167,7 @@ func (host *goLanguageHost) GetRequiredPlugins(ctx context.Context,
 
 	cmd.Env = os.Environ()
 	cmd.Env = append(cmd.Env, "PULUMI_PLUGINS=true")
+	cmd.Stderr = os.Stderr
```

Comment:
Do we intentionally not want to redirect SdOut as well?

Reply:
from the description: 

> Note that we're already capturing stdout in line 172, so we don't redirect it here.

If you try to capture stdout, `cmd.Output()` will return an error saying that stdout is already bound.